src/sentry/features/__init__.py

```diff
@@ -227,6 +227,7 @@
 default_manager.add("projects:similarity-view", ProjectFeature)
 default_manager.add("projects:similarity-view-v2", ProjectFeature)
 default_manager.add("projects:suspect-resolutions", ProjectFeature, True)
+default_manager.add("projects:transaction-name-cluster", ProjectFeature, True)
```

Comment:
I'm wondering if this should be an org feature instead of a project feature. Although the clusterer will run for projects, having projects with different behavior under the same org can be confusing. Adding/removing orgs instead of projects may also be easier to collect feedback.